notebooks/community/sdk/SDK_BigQuery_Custom_Container_Training.ipynb

```diff
@@ -58,8 +58,15 @@
       },
```

Comment:
<p>remove 'will'</p><p> </p><p>we -&gt; you</p><p> </p><p>Big Query -&gt; BigQuery</p><p> </p><p>Use sentence case in title</p><p> </p><p>Describe the params used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>